src/scenes/ProgressReports/Detail/ProgressReportDrawer/InstructionsDialog.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  Typography,
+} from '@mui/material';
+
+export const InstructionsDialog = ({
+  open,
+  onClose,
+}: {
+  open: boolean;
+  onClose: () => void;
+}) => {
```

Comment:
I would just spread through all the dialog props here
```suggestion
export const InstructionsDialog = (props: DialogProps) => {
```